docs/UserGuide.md

```diff
@@ -743,7 +747,7 @@ instructions.
 
 2.  Reference the Swagger documentation located in the /docs folder of
     the repo to view the specifications for the API call. If needed,
-    paste the YAML file into http://editor.swagger.io to see a rendered
+    paste the YAML file into <http://editor.swagger.io> to see a rendered
```

Comment:
The numbered lists in section 7.5.* are not rendering correctly. They are supposed to be nested and indented.